assets/css/main.scss

```diff
@@ -209,16 +209,21 @@ footer {
     .thumbnail img {
         background-color: white;
         border-radius: 20px;
-        border: 1px solid $gray;
+        border: 1px solid #ddd;
```

Comment:
I chose a variable because: 1. it is better to rely on variables rather than hardcoded values in sass, and 2. it is nicer/more consistent to have a small palette.

Reply:
ok cool, I will add it as a variable then :)